ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -352,6 +362,9 @@ spec:
       - hostPath:
           path: "/var/log/oauth-apiserver"
         name: auditlogs-oauth-apiserver
+      - hostPath:
+          path: "${SHARED_DIR}"
```

Comment:
Why are we mounting `${SHARED_DIR}`? This needs to be a directory on the nodes, not artifacts dir path

Reply:
Thanks for pointing out the problem :) 
I have some difficulty mounting the volume where the logs are generated directly to prom-tail container. The volume where the logs are written is a type "emptyDir" volume that dies with the pod. During the must-gather step, these logs are extracted to the ${ARTIFACT_DIR} directory. So I combine the logs into a single file and pass it to the loki-promtail container using ${SHARED_DIR} directory.
I'm not sure this works because loki container starts before the test starts and the must-gather step executes after the test. I wonder how to send these logs on temporary volume to loki?

Reply:
>The volume where the logs are written is a type "emptyDir" volume that dies with the pod

It needs to be written on the node so that Promtail could pick them up. It cannot extract them from must-gather or other steps

Reply:
Thanks, I'm going to check if the log can be moved to a persistent volume.